Classes/KIFUITestActor.m

```diff
@@ -622,22 +622,38 @@ - (void)selectPickerValue:(NSArray *)pickerColumnValues pickerType:(KIFPickerTyp
         }
         // Find the picker view
         UIPickerView *pickerView = nil;
+        NSInteger firstRowToSearchIndex = 0;
         switch (pickerType)
         {
             case KIFUIDatePicker:
+            {
                 pickerView = [[[[UIApplication sharedApplication] datePickerWindow] subviewsWithClassNameOrSuperClassNamePrefix:@"UIPickerView"] lastObject];
                 KIFTestCondition(pickerView, error, @"No picker view is present");
+                
+                // If minimum date limit exist, search the value from minimum and beyond.
+                UIDatePicker *datePicker = (UIDatePicker *)pickerView;
+                if (datePicker.datePickerMode == UIDatePickerModeDateAndTime
+                    && datePicker.minimumDate != nil) {
+                    
+                    [datePicker setDate:datePicker.minimumDate];
```

Comment:
Sounds great. 
I was tring to think of a more general solution than my fix. 
your suggestion looks great. Gives more control back to the caller.

I will implement it and push. should I open another pull request or can I use this one?